recipes/posthog/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "posthog" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: be7caf0625c6efdc6cb7052d4026a13c3d94df4a2662b90945de02d2cfc21bd4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7.0,<4.0.0
+    - pip
+  run:
+    - python >=3.7.0,<4.0.0
+    - requests >=2.7.0,<3.0.0
+    - six >=1.5.0,<2.0.0
+    - monotonic >=1.5.0,<2.0.0
+    - backoff >=1.6.0,<2.0.0
```

Comment:
I guess the exact pin is a bug upstream. Can you open a PR/issue upstream and apply a patch if you're going to loosen the requirement so you can keep the `pip check` test?

Reply:
merged upstream in https://github.com/PostHog/posthog-python/pull/36 but not yet released to pypi - can we publish in the meantime as this is holding up an update to our main package?